azerothcore/azerothcore-wotlk
# fix(DB/SAI): Harbinger of the Raven falling instead of flying.

## Changes Proposed:
- Fixed sometimes despawning instantly when he lands
- Fixed flying
- Attack invoker, not closest player
- Added OOC (15s) & on evade despawn. 

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/15002
- Closes https://github.com/chromiecraft/chromiecraft/issues/4991

## SOURCE:

https://youtu.be/zTxl9XJJ36I?t=184
## Tests Performed:

- tested


## How to Test the Changes:


.go zonexy 64.47 33.1 3829
.quest add 10567
.add 30704 6
Use the Ruaan'ok Claw

See if he correctly flies down and despawns shortly after if not engaged in combat OR he evades.
See if he attacks the invoker.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
